### PR TITLE
UI Improvement

### LINKED ISSUE #105: Change App Colour Theme
Change the colour theme from black to one that uses dark blue `#102c44` (tentative) as the primary colour.

### LINKED ISSUE #129: Change product logo

### PR DESCRIPTION
**Pull request by florentianayuwono**
- Logo changed
- Color scheme not yet finalized (need feedbacks)
- Need help figuring out the css that makes the grayish part persistent :") literally removed all instances of the gray but still present 
- Placeholder square icon for todo, will be replaced by calendar and dot icon
image
image

### CODE REVIEW

**Comment by florentianayuwono** [comment]
Update: gray area cause found in the `Extension.css`. Edited, but now left the thin border thing.

image

File: src/main/java/soconnect/ui/UiManager.java [hunk 1]
```diff
@@ -20,7 +20,7 @@ public class UiManager implements Ui {
     public static final String ALERT_DIALOG_PANE_FIELD_ID = "alertDialogPane";
 
     private static final Logger logger = LogsCenter.getLogger(UiManager.class);
-    private static final String ICON_APPLICATION = "/images/SoConnect_32.png";
+    private static final String ICON_APPLICATION = "/images/SoConnect Logo.png";
```
Comment (PangKuangWei) [inline src/main/java/soconnect/ui/UiManager.java]: I am a bit worried that having whitespace in filenames might cause issues somewhere.

**Review by PangKuangWei — COMMENTED** [review]
Great job on upgrading the GUI and the Logo looks great. We need to extensively check for any remnants of the dark theme and ensure there is no background and text fill with the same colour/similar colour.

File: src/main/java/soconnect/ui/UiManager.java (hunk 1 above)
Comment (florentianayuwono) [reply src/main/java/soconnect/ui/UiManager.java]: So far it's rendering the logo okay. But can, I will change it :D

Comment (PangKuangWei) [reply src/main/java/soconnect/ui/UiManager.java]: I'm just worried for what happens in MacOS and Linux.

**Review by PangKuangWei — APPROVED** [review]
LGTM

**Pull request merged**